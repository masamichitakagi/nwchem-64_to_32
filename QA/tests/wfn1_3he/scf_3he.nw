echo
start wfn1_he_dat

geometry
  he 0 0 0
end
basis
He    S
     38.4216340              1.0000000
He    S
      5.7780300              1.0000000
He    S
      1.2417740              1.0000000
He    S
      0.2979640              1.0000000
end

dft
  odft
  mult 3
  xc hfexch
end

task dft energy

dft
  odft
  mult 3
  xc hfexch
  smear 1.5
end

task dft energy

