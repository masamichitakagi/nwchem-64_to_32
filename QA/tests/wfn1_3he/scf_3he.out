 argument  1 = scf_3he.nw



============================== echo of input deck ==============================
echo
start wfn1_he_dat

geometry
  he 0 0 0
end
basis
He    S
     38.4216340              1.0000000
He    S
      5.7780300              1.0000000
He    S
      1.2417740              1.0000000
He    S
      0.2979640              1.0000000
end

dft
  odft
  mult 3
  xc hfexch
end

task dft energy

dft
  odft
  mult 3
  xc hfexch
  smear 1.5
end

task dft energy

================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.6
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2015
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = LDC-153112.home
    program         = ../../../bin/MACX64/nwchem
    date            = Sun Mar 19 00:13:38 2017

    compiled        = Sun_Mar_19_00:01:50_2017
    source          = /Users/vandam/nwchem-wfn1-dev
    nwchem branch   = Development
    nwchem revision = 28658
    ga revision     = 10724
    input           = scf_3he.nw
    prefix          = wfn1_he_dat.
    data base       = ./wfn1_he_dat.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107196 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428797 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)



                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 he                   2.0000     0.00000000     0.00000000     0.00000000

      Atomic Mass 
      ----------- 

      he                 4.002600


 Effective nuclear repulsion energy (a.u.)       0.0000000000

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000


            XYZ format geometry
            -------------------
     1
 geometry
 he                    0.00000000     0.00000000     0.00000000

                      Basis "ao basis" -> "" (cartesian)
                      -----
  He (Helium)
  -----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.84216340E+01  1.000000

  2 S  5.77803000E+00  1.000000

  3 S  1.24177400E+00  1.000000

  4 S  2.97964000E-01  1.000000



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 He                      user specified              4        4   4s


  xc_inp: hfexch multiplicative factor not found.

                                 NWChem DFT Module
                                 -----------------




 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 He                      user specified              4        4   4s


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  spin polarized.
          No. of atoms     :     1
          No. of electrons :     2
           Alpha electrons :     2
            Beta electrons :     0
          Charge           :     0
          Spin multiplicity:     3
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  30
          AO basis - number of functions:     4
                     number of shells:     4
          Convergence on energy requested:  1.00D-06
          Convergence on density requested:  1.00D-05
          Convergence on gradient requested:  5.00D-04

              XC Information
              --------------
                     Hartree-Fock (Exact) Exchange  1.000          

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL):  1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho:  1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul:  1.00D-08


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:          -2.85515891

      Non-variational initial energy
      ------------------------------

 Total energy =      -2.855160
 1-e energy   =      -3.882067
 2-e energy   =       1.026907
 HOMO         =      -0.914127
 LUMO         =       1.162113

 Initial wavefunction

 global array: alpha evecs[1:4,1:4],  handle: -995 

            1           2           3           4  
       ----------- ----------- ----------- -----------
   1      -0.02377    -0.02804    -0.00880     1.25399
   2      -0.15468    -0.06909    -1.41020    -1.01810
   3      -0.46963    -1.17810     1.46027     0.51865
   4      -0.51359     1.29978    -0.54473    -0.17311
   Time after variat. SCF:      0.2
   Time prior to 1st pass:      0.2

 #quartets = 5.500D+01 #integrals = 5.500D+01 #direct =  0.0% #cached =100.0%


 Integral file          = ./wfn1_he_dat.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  34250
 No. of bits per label  =      8        No. of bits per value  =     64


           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       12.98            12975548
          Stack Space remaining (MW):       13.11            13107036

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1     -1.5431417028 -1.54D+00  4.57D-02  1.77D-01     0.3
                                                     0.00D+00  0.00D+00
 d= 0,ls=0.0,diis     2     -1.5547417690 -1.16D-02  6.04D-03  3.14D-03     0.3
                                                     0.00D+00  0.00D+00
 d= 0,ls=0.0,diis     3     -1.5549439626 -2.02D-04  3.84D-05  5.59D-07     0.3
                                                     0.00D+00  0.00D+00
  Resetting Diis
 d= 0,ls=0.0,diis     4     -1.5549439875 -2.49D-08  1.54D-06  2.62D-10     0.3
                                                     0.00D+00  0.00D+00


         Total DFT energy =       -1.554943987518
      One electron energy =       -2.139175306136
           Coulomb energy =        1.624681812085
    Exchange-Corr. energy =       -1.040450493467
 Nuclear repulsion energy =        0.000000000000

 Numeric. integr. density =        0.000000000000

     Total iterative time =      0.1s



                    DFT Final Alpha Molecular Orbital Analysis
                    ------------------------------------------

 Vector    1  Occ=1.000000D+00  E=-1.409819D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 2.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.618661  1 He s                  4      0.304989  1 He s          
     2      0.202821  1 He s                  1      0.028357  1 He s          

 Vector    2  Occ=1.000000D+00  E= 4.391049D-01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 9.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      1.351088  1 He s                  3     -1.040401  1 He s          
     2     -0.083031  1 He s          

 Vector    3  Occ=0.000000D+00  E= 8.095999D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 2.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3     -1.507805  1 He s                  2      1.402055  1 He s          
     4      0.575428  1 He s          

 Vector    4  Occ=0.000000D+00  E= 6.197132D+01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      1.253988  1 He s                  2     -1.019858  1 He s          
     3      0.520780  1 He s                  4     -0.174146  1 He s          


                     DFT Final Beta Molecular Orbital Analysis
                     -----------------------------------------

 Vector    1  Occ=0.000000D+00  E=-1.994269D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 2.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.626083  1 He s                  4      0.302481  1 He s          
     2      0.195501  1 He s                  1      0.028408  1 He s          

 Vector    2  Occ=0.000000D+00  E=-1.475096D-01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 9.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      1.339618  1 He s                  3     -1.011412  1 He s          
     2     -0.110329  1 He s          

 Vector    3  Occ=0.000000D+00  E= 6.318385D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 2.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3     -1.522567  1 He s                  2      1.397903  1 He s          
     4      0.602196  1 He s          

 Vector    4  Occ=0.000000D+00  E= 5.938939D+01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      1.253959  1 He s                  2     -1.024388  1 He s          
     3      0.526041  1 He s                  4     -0.176605  1 He s          


   alpha - beta orbital overlaps 
   ----------------------------- 


   alpha      1      2      3      4
    beta      1      2      3      4
 overlap   1.000  1.000  1.000  1.000

     --------------------------
     Expectation value of S2:  
     --------------------------
      <S2> =      2.0000 (Exact =     2.0000)


 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.2s     wall:        0.2s


                                NWChem Input Module
                                -------------------


  xc_inp: hfexch multiplicative factor not found.

                                 NWChem DFT Module
                                 -----------------




 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 He                      user specified              4        4   4s


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  spin polarized.
          No. of atoms     :     1
          No. of electrons :     2
           Alpha electrons :     2
            Beta electrons :     0
          Charge           :     0
          Spin multiplicity:     3
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  30
          AO basis - number of functions:     4
                     number of shells:     4
          Convergence on energy requested:  1.00D-06
          Convergence on density requested:  1.00D-05
          Convergence on gradient requested:  5.00D-04

              XC Information
              --------------
                     Hartree-Fock (Exact) Exchange  1.000          

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL):  1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 

          Smearing applied:  0.15D+01 (hartree)

      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho:  1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul:  1.00D-08


 Loading old vectors from job with title :



   Time after variat. SCF:      0.3
 sigma=  0.15D+01 eFermi=  0.00000 T*S= -1.05E+00 nel(1)=    2.00 nel(2)=    0.00

   Time prior to 1st pass:      0.3

 #quartets = 5.500D+01 #integrals = 5.500D+01 #direct =  0.0% #cached =100.0%


 Integral file          = ./wfn1_he_dat.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  34250
 No. of bits per label  =      8        No. of bits per value  =     64

 sigma=  0.15D+01 eFermi=  4.26759 T*S= -1.25E-03 nel(1)=    2.00 nel(2)=    0.00

  Resetting Diis

           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       12.98            12975628
          Stack Space remaining (MW):       13.11            13107004

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1     -1.5561986088 -1.56D+00  1.69D-04  5.63D-15     0.4
                                                     0.00D+00  0.00D+00
 sigma=  0.15D+01 eFermi=  4.26759 T*S= -1.26E-03 nel(1)=    2.00 nel(2)=    0.00

 d= 0,ls=0.0,diis     2     -1.5550242561  1.17D-03  6.16D-07  1.24D-09     0.4
                                                     0.00D+00  0.00D+00
 sigma=  0.15D+01 eFermi=  4.26759 T*S= -1.26E-03 nel(1)=    2.00 nel(2)=    0.00

 d= 0,ls=0.0,diis     3     -1.5550238318  4.24D-07  6.37D-08  1.97D-12     0.4
                                                     0.00D+00  0.00D+00


         Total DFT energy =       -1.553768785030
      One electron energy =       -2.138183263494
           Coulomb energy =        1.624817068766
    Exchange-Corr. energy =       -1.040402590301
 Nuclear repulsion energy =        0.000000000000

 Numeric. integr. density =        0.000000000000

     Total iterative time =      0.1s



                    DFT Final Alpha Molecular Orbital Analysis
                    ------------------------------------------

 Vector    1  Occ=1.000000D+00  E=-1.409734D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 2.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.618642  1 He s                  4      0.305015  1 He s          
     2      0.202818  1 He s                  1      0.028357  1 He s          

 Vector    2  Occ=9.998466D-01  E= 4.392047D-01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 9.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      1.351082  1 He s                  3     -1.040413  1 He s          
     2     -0.083035  1 He s          

 Vector    3  Occ=1.534745D-04  E= 8.095899D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 2.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3     -1.507804  1 He s                  2      1.402055  1 He s          
     4      0.575428  1 He s          

 Vector    4  Occ=0.000000D+00  E= 6.197151D+01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      1.253988  1 He s                  2     -1.019858  1 He s          
     3      0.520780  1 He s                  4     -0.174146  1 He s          


                     DFT Final Beta Molecular Orbital Analysis
                     -----------------------------------------

 Vector    1  Occ=0.000000D+00  E=-1.994269D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 2.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.626083  1 He s                  4      0.302481  1 He s          
     2      0.195501  1 He s                  1      0.028408  1 He s          

 Vector    2  Occ=0.000000D+00  E=-1.475096D-01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 9.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      1.339618  1 He s                  3     -1.011412  1 He s          
     2     -0.110329  1 He s          

 Vector    3  Occ=0.000000D+00  E= 6.318385D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 2.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3     -1.522567  1 He s                  2      1.397903  1 He s          
     4      0.602196  1 He s          

 Vector    4  Occ=0.000000D+00  E= 5.938939D+01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      1.253959  1 He s                  2     -1.024388  1 He s          
     3      0.526041  1 He s                  4     -0.176605  1 He s          


   alpha - beta orbital overlaps 
   ----------------------------- 


   alpha      1      2      3      4
    beta      1      2      3      4
 overlap   1.000  1.000  1.000  1.000

     --------------------------
     Expectation value of S2:  
     --------------------------
      <S2> =      2.0000 (Exact =     2.0000)


 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.1s     wall:        0.1s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  327      327      829      530      181        0        0      110     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             8.01e+04 3.93e+04 2.32e+04 0.00e+00 0.00e+00 8.80e+02
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 8320 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        19	        20
	current total bytes		         0	         0
	maximum total bytes		   1053208	  22509928
	maximum total K-bytes		      1054	     22510
	maximum total M-bytes		         2	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
       S. A. Fischer, S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen,
    V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler, V. Konjkov, T. Risthaus,
       M. Malagoli, A. Marenich, A. Otero-de-la-Roza, J. Mullin, P. Nichols,
      R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari, M. Williamson,
      R. J. Harrison, J. R. Rehr, M. Dupuis, D. Silverstein, D. M. A. Smith,
            J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
   H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols, K. Tsemekhman,
    K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski, T. Clark, D. Clerc,
   H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
   A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                               A. T. Wong, Z. Zhang.

 Total times  cpu:        0.4s     wall:        0.4s
